tests/mbed/host/host.c

```diff
@@ -129,7 +132,7 @@ int main(int argc, const char* argv[])
 
     printf("=== %s: %s\n", argv[0], argv[1]);
 
-    strcpy(temp, argv[1]);
+    strcpy_s(temp, TEMP_LEN, argv[1]);
```

Comment:
sizeof(temp) instead of TEMP_LEN.  Avoid using constants as the 2nd argument to strcpy_s throughout.

Reply:
Just tried changing all the second argument to `sizeof(...)` but all the mbed tests failed. So I leave this as it is for now.

Reply:
huh. sizeof(temp) should be the same as TEMP_LEN. What exactly fails?

Reply:
I just tried again. It seems that if I only change the one in main() it will be fine. Otherwise ctest will fail with an exception code 0xc0000409.

Reply:
This is the only place where the size of the buffer is known by the compiler. I commented elsewhere on what you should do instead